.github/workflows/main.yml

```diff
@@ -91,20 +93,21 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         working-directory: setuptools/tests/config
         run: python -m downloads.preload setupcfg_examples.txt
+      - name: Workaround for unreleased PyNaCl (pyca/pynacl#805)
+        if: contains(matrix.python, 'pypy')
+        run: echo "SETUPTOOLS_ENFORCE_DEPRECATION=0" >> $GITHUB_ENV
+      - name: Install tox
+        run: python -m pip install tox
```

Comment:
I just moved this part around such that tox would be installed before pre-building distributions[^1].

[^1]: The pre-built distributions were previously introduced to be used with virtualenv fixtures, it is also used by downstream packagers.